x-pack/plugins/security_solution/common/api/timeline/pinned_events/pinned_events_route.schema.yaml

```diff
@@ -51,11 +51,22 @@ paths:
                     required: [persistPinnedEventOnTimeline]
                     properties:
                       persistPinnedEventOnTimeline:
-                        allOf:
-                          - $ref: '../model/components.schema.yaml#/components/schemas/PinnedEvent'
-                          - type: object
-                            properties:
-                              code:
-                                type: number
-                              message:
-                                type: string
+                        $ref: '#/components/schemas/PersistPinnedEventResponse'
+
+components:
+  schemas:
+    PersistPinnedEventResponse:
+      oneOf:
+        - allOf:
+            - $ref: '../model/components.schema.yaml#/components/schemas/PinnedEvent'
+            - $ref: '#/components/schemas/PinnedEventBaseResponseBody'
+        - type: object
+          nullable: true
```

Comment:
In the current version of OpenAPI we cannot simply return `null` here. The workaround is to return an empty nullable object (`{} || null`)